ui/src/components/flows/tasks/TaskList.vue

```diff
@@ -5,7 +5,7 @@
             :elements="items"
             :section
             :block-schema-path="[blockSchemaPath, 'properties', root, 'items'].join('/')"
-            @remove="(yaml) => flowStore.flowYaml = yaml"
+            @remove="removeItems"
```

Comment:
NIT: Since we only remove item can we call it `removeItem`?

```suggestion
            @remove="removeItem"
```